README.adoc

```diff
@@ -18,171 +18,14 @@
 //
 image:https://img.shields.io/maven-central/v/org.eclipse.microprofile.context-propagation/microprofile-context-propagation-api.svg[link="http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.eclipse.microprofile.context-propagation%22%20AND%20a%3A%22microprofile-context-propagation-api%22"]
 image:https://javadoc.io/badge/org.eclipse.microprofile.context-propagation/microprofile-context-propagation-api.svg[ link="https://javadoc.io/doc/org.eclipse.microprofile.context-propagation/microprofile-context-propagation-api"]
+image:https://img.shields.io/badge/Specification-1.0-blue.svg[link="https://download.eclipse.org/microprofile/microprofile-context-propagation-1.0/microprofile-context-propagation.html"]
 image:https://badges.gitter.im/eclipse/microprofile-concurrency.svg[link="https://gitter.im/eclipse/microprofile-concurrency"]
 
 [[microprofile-context-propagation]]
 = MicroProfile Context Propagation
 
 :toc:
 
-* Status: *new*
-* Decision Notes:
-https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!topic/microprofile/jKFu-IS_U90[Discussion
-thread topic with background and covering the design]
```

Comment:
Thanks for catching the out-of-date material and removing it.  I agree with removing the above, however, removing what follows after this would leave us with an empty doc on the main page, so I think it would be better to just update more specific out-of-date wording.  I'll add some review comments below attempting to do that.

Reply:
I thought that initially too, except we do have a link to the rendered spec so it's easy to get to the info, and whatever we duplicate here will always end up out-of-date.

I think this file should concentrate on things that are not in the spec.

A short intro of a single paragraph should be fine, though.